### PR TITLE
Implement endpoint to list available stacks and endpoint to update app stack

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: apps.go [hunk 1]
```diff
@@ -216,6 +217,33 @@ func (c *Client) AppsTransfer(name string, email string) (*App, error) {
...
+	res, err := req.Do()
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	var appRes *AppResponse
+	err = ParseJSON(res, &appRes)
+	if err != nil {
+		return nil, errgo.Mask(err, errgo.Any)
```
Comment (EtienneM) [inline apps.go]: Should we use `errgo.Mask` in new code? Don't we want to use at least `Notef`?

File: stacks.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+func (c *Client) StacksList() ([]Stack, error) {
+	req := &APIRequest{
+		Client:   c,
+		Endpoint: "/features/stacks",
+	}
+
+	res, err := req.Do()
+	if err != nil {
+		return nil, errgo.Mask(err, errgo.Any)
```
Comment (EtienneM) [inline stacks.go]: Same here

Comment (Soulou) [reply stacks.go]: 100% right

**Comment by Soulou** [comment]
Updated

**Pull request merged**